doc/langref/destructuring_to_existing.zig

```diff
@@ -0,0 +1,17 @@
+const print = @import("std").debug.print;
+
+pub fn main() void {
+    var x: u32 = undefined;
+    var y: u32 = undefined;
+    var z: u32 = undefined;
+
+    const tuple = .{ 1, 2, 3 };
+
+    x, y, z = tuple;
+
+    print("x = {}\n", .{x});
+    print("y = {}\n", .{y});
+    print("z = {}\n", .{z});
```

Comment:
Probably worth combining these prints into a single line, just to make the doc a bit shorter.